db/re/item_db.txt

```diff
@@ -11534,14 +11597,139 @@
+23837,Bs_Item_M_S_15,Survival SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23837; },{},{}
+23838,Bs_Item_M_S_28,Toy Sringe SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23838; },{},{}
+23839,Bs_Item_M_S_29,Indigor Rear Ribbon SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23839; },{},{}
+23840,Bs_Item_M_S_31,Magical Booster SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23840; },{},{}
+23841,Bs_Item_M_S_32,Rosario's Necklace SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23841; },{},{}
+23842,Bs_Item_M_S_33,Spirit Crown SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23842; },{},{}
+23843,Bs_Item_M_S_36,Guard's Cap SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23843; },{},{}
+23844,Bs_Item_M_S_37,Bandit's Hood SynthesisBox,11,,,1,,0,,0,0xFFFFFFFF,63,2,,0,,,,{ synthesisui 23844; },{},{}
+23845,Bs_Item_M_S_38,Angel's Blessing SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23845; },{},{}
+23846,Bs_Item_M_S_39,Rabbit Magic Hat SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23846; },{},{}
+23847,Bs_Item_M_S_40,Anubis Helm SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23847; },{},{}
+23848,Bs_Item_M_S_45,Imp Hat SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23848; },{},{}
+23849,Bs_Item_M_S_46,Red Marching Hat SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23849; },{},{}
+23850,Bs_Item_M_S_47,Ifrit Mask SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23850; },{},{}
+23851,Bs_Item_M_S_48,Incarnation Of Morocc Doll SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23851; },{},{}
+23852,Bs_Item_M_S_49,Samambaia SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23852; },{},{}
+23853,Bs_Item_M_S_50,Chick Hat SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23853; },{},{}
+23854,Bs_Sha_M_S_2,Class Shadow SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23854; },{},{}
+23855,Bs_Sha_M_S_5,Spell Shadow SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23855; },{},{}
+23856,Bs_Sha_M_S_6,Size Shadow SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23856; },{},{}
+23857,Bs_Sha_M_S_7,Race Shadow SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23857; },{},{}
+23858,Bs_Sha_M_S_8,Stability Shadow SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23858; },{},{}
+23859,Bs_Sha_M_S_13,Special Shadow SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23859; },{},{}
+23860,Bs_Sha_M_S_15,Physical Shadow SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23860; },{},{}
+23861,Bs_Sha_M_S_16,Magical Shadow SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23861; },{},{}
+23862,Bs_Sha_M_S_23,EXP Shadow SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23862; },{},{}
+23863,Bs_Item_M_S_5,Mad Bunny SynthesisBox,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23863; },{},{}
```

Comment:
After using this item with correct requirements you can't get any items unless the scroll goes back on your pocket.

Reply:
Maybe you don't have the items in your itemdb or it's not in the emulator and grf lapineddukddakbox